docs/02-get-started/03-deploy-expose-microservice.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Deploy and expose a microservice
+---
+
+You already know how to [deploy](02-deploy-expose-function.md#create-a-function) and [expose a Function](02-deploy-expose-function.md#expose-the-function). Let's now do the same with a container microservice.
+We'll use the Kyma example [`orders-service`](https://github.com/kyma-project/examples/blob/master/orders-service/README.md) for this.
+
+## Deploy the microservice
+
+First, let's create a Deployment that provides the microservice definition and lets you run it on the cluster. 
+
+<div tabs name="Create a microservice Deployment" group="deploy-expose-microservice">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+cat <<EOF | kubectl apply -f -
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: orders-service
+    namespace: default
+    labels:
+      app: orders-service
+      example: orders-service
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: orders-service
+        example: orders-service
+    template:
+      metadata:
+        labels:
+          app: orders-service
+          example: orders-service
+      spec:
+        containers:
+          - name: orders-service
+            image: "eu.gcr.io/kyma-project/develop/orders-service:e8175c63"
+            imagePullPolicy: IfNotPresent
+            resources:
+              limits:
+                cpu: 20m
+                memory: 32Mi
+              requests:
+                cpu: 10m
+                memory: 16Mi
+            env:
+              - name: APP_PORT
+                value: "8080"
+              - name: APP_REDIS_PREFIX
+                value: "REDIS_"
+EOF
+```
+
+To check that the Deployment was created successfully, run: 
+```bash
+kubectl get deployment orders-service -o=jsonpath="{.status.readyReplicas}"
+```
+
+The operation was successful if the returned number of **readyReplicas** is `1`.
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. From the left navigation, go to **Deployments**.
+2. Click on **Create Deployment +**.
+3. Provide the following parameters:
+    - **Name**: `orders-service`
+    - **Labels**: `app=orders-service` and `example=orders-service`
+    - **Docker image**: `eu.gcr.io/kyma-project/develop/orders-service:e8175c63`
+   
+    _Optionally_, to save resources, modify these parameters:
+    - **Memory requests**: `10Mi`
+    - **CPU requests**: `16m`
+    - **Memory limits**: `32Mi`
+    - **CPU limits**: `20m`
+   
+    Leave the checkbox to create a Service checked and skip the next Section.
```

Comment:
And I see this has now changed in the Busola and needs to be modified, as we have three views: simple, advanced, and YAML, and only **Name** and **Image** are visible in the simple view, and in the advanced view, for the labels, keys and values are provided in separate fields. 
![image](https://user-images.githubusercontent.com/53601578/133768515-a9efe8ef-abe9-4ea1-ae68-c327c338c6b1.png)
![image](https://user-images.githubusercontent.com/53601578/133768530-e1cb549b-c410-48ba-bf9b-e418f793f13e.png)
![image](https://user-images.githubusercontent.com/53601578/133768583-eb93831f-2b6f-4426-9818-35a0da1f6786.png)

Reply:
Fixed in https://github.com/kyma-project/kyma/pull/12128